h/auth/tokens.py

```diff
@@ -27,6 +28,9 @@ def __init__(self, token_model):
         self.expires = token_model.expires
         self.userid = token_model.userid
 
+        # Associates the userid with a given transaction/web request.
```

Comment:
This seemed like the appropriate place as it is called for each request regardless of how the request was issued.  If there's somewhere else more appropriate lmk!